### PR TITLE
Feature ck dispaly short reference

### PR DESCRIPTION
**Pull request by christopher-kundrat**

### CODE REVIEW

File: org/Hibachi/HibachiTags/HibachiEntityDetailGroup.cfm [hunk 1]
```diff
@@ -106,6 +106,9 @@
 												<cfif attributes.object.hasProperty('remoteID')>
 													<hb:HibachiPropertyDisplay object="#attributes.object#" property="remoteID" edit="#iif(request.context.edit && attributes.hibachiScope.setting('globalRemoteIDEditFlag'), true, false)#" />
 												</cfif>
+												<cfif len( attributes.object.getshortReferenceID() )>
```
Comment (gregmoser) [inline org/Hibachi/HibachiTags/HibachiEntityDetailGroup.cfm]: Small Casing issue... should be ``` attributes.object.getShortReferenceID() ``` fine to merge the way it is though

**Comment by christopher-kundrat** [comment]
Updated to the proper casing

**Comment by rmarchandTen24** [comment]
change title to use an rb key entity.define.shortreferenceid

**Comment by christopher-kundrat** [comment]
Updated rb key

**Pull request merged**